packages/chart-advisor/src/advisor/advice-pipeline/interface.ts

```diff
@@ -2,6 +2,9 @@ import { Purpose } from '@antv/ckb';
 import { AntVSpec } from '@antv/antv-spec';
 import { Preferences } from '../../ruler';
 
+export type DataRow = Record<string, any>;
```

Comment:
I believe we could put these in DW and import them here? @soundquiet @pddpd

Reply:
I saw definitions such as `FrameData` in DW but have to import through `@antv/data-wizard/lib/dataset`, maybe export them? @pddpd

Reply:
About `FrameData`? Record<string, any>/{[key: string]: any;} is a part of it,  can it meet ur requirement? @soundquiet

Reply:
Yes, I tried to replace `DataRow` as `FrameData` and it works.
but currently CA can only support `Record<string, any>`, because antv-spec cannot handle other types of data for now. 

Is that ok to define a variable as `FrameData` but I only use it as `Record<string, any>`?

@pddpd @neoddish

Reply:
If u only want to use `Record<string, any>`, I think u can use it directly. It's a normal type definition.

Reply:
OK, I would keep`Record<string, any>` in this PR